code/modules/surgery/organs_internal.dm

```diff
@@ -110,6 +119,27 @@
 			user.visible_message("[user] starts transplanting \the [tool] into [target]'s [parse_zone(target_zone)].", \
 			"You start transplanting \the [tool] into [target]'s [parse_zone(target_zone)].")
 
+	else if(implement_type in implements_clean)
+		current_type = "clean"
+
+		if(!istype(tool, /obj/item/weapon/reagent_containers/))
```

Comment:
should be `/obj/item/weapon/reagent_containers` not `/obj/item/weapon/reagent_containers/`